src/test/unit_tests/test_nb_native_fs.js

```diff
@@ -47,6 +65,11 @@ mocha.describe('nb_native fs', function() {
                 _.omit(res, 'xattr', 'mtimeNsBigint', 'atimeNsBigint', 'ctimeNsBigint'), // we need to remove xattr, mtimeSec, mtimeNsec from fs_napi response as the node JS stat doesn't return them
                 _.omitBy(res2, v => typeof v === 'function'),
             );
+
+            assert.notDeepStrictEqual(
+                _.omit(res, 'xattr', 'mtimeNsBigint', 'atimeNsBigint', 'ctimeNsBigint'), // we need to remove xattr, mtimeSec, mtimeNsec from fs_napi response as the node JS stat doesn't return them
```

Comment:
If find this comment confusing since the properties are not the same (I know it was copied from the code above).
```suggestion
                _.omit(res, 'xattr', 'mtimeNsBigint', 'atimeNsBigint', 'ctimeNsBigint'), // we need to remove those properties from fs_napi response as the node JS stat doesn't return them
```